tests/test_kernelization.py

```diff
@@ -0,0 +1,191 @@
+from __future__ import annotations
+import mis.pipeline.kernelization as ker
+import networkx as nx
+
+
+class InitKernelization(ker.Kernelization):
+    """
+    Utility class that initializes an instance of Kernelization
+    during `__init__` instead of during the call to `preprocess`.
+    """
+
+    def __init__(self, graph: nx.Graph):
+        self.reset(graph)
+
+
+def test_is_subclique() -> None:
+    test_instance = InitKernelization(nx.complete_graph(5))
+    test_instance.kernel.add_node(30)
+    assert test_instance.is_subclique([0, 1, 2, 3])
+    assert not test_instance.is_subclique([0, 1, 2, 3, 30])
```

Comment:
Do I undersand that any list of nodes that contains the node labelled 30 is not a subclique ? In that case, does it make sense to draw randomly node labels inside and outside the complete graph to test this ?

Reply:
Not sure I understand.